tests/targets/test_android.py

```diff
@@ -52,25 +52,107 @@ def patch_platform(platform):
 
 
 class TestTargetAndroid:
+
     @staticmethod
     def default_specfile_path():
         return os.path.join(os.path.dirname(buildozer_module.__file__), "default.spec")
 
     def setup_method(self):
-        """Creates a temporary spec file containing the content of the default.spec."""
-        self.specfile = tempfile.NamedTemporaryFile(suffix=".spec", delete=False)
-        default_spec = codecs.open(self.default_specfile_path(), encoding="utf-8")
-        self.specfile.write(default_spec.read().encode("utf-8"))
-        self.specfile.close()
-        self.buildozer = Buildozer(filename=self.specfile.name, target="android")
-        self.target_android = TargetAndroid(self.buildozer)
+        """
+        Create a temporary directory that will contain the spec file and will
+        serve as the root_dir.
+        """
+        self.temp_dir = tempfile.TemporaryDirectory()
 
     def tear_method(self):
-        """Deletes the temporary spec file."""
-        os.unlink(self.specfile.name)
+        """
+        Remove the temporary directory created in self.setup_method.
+        """
+        self.temp_dir.cleanup()
+
+    def init_target(self, options={}):
```

Comment:
default mutable arguments is not recommended: https://docs.python-guide.org/writing/gotchas/

Reply:
Thank you, fixed it (: